tests/l2/README.md

```diff
@@ -28,6 +28,10 @@ $ oc logs intel-sgx-job-4tnh5
 ```
 ### Verify Intel® Data Center GPU provisioning
 This workload runs [clinfo](https://github.com/Oblomov/clinfo) utilizing the i915 resource from GPU provisioning and displays the related GPU information.
+* Before deploying any workload, please ensure to set access for ```/dev/``` folder on the GPU node, with the command below:
```

Comment:
Can we change it to:

To work around [issue](https://github.com/intel/intel-technology-enabling-for-openshift/issues/107), please run the below command on the node with the GPU where your workload will run.

Reply:
done